numpy/core/numeric.py

```diff
@@ -2089,15 +2089,12 @@ def indices(dimensions, dtype=int):
     """
     dimensions = tuple(dimensions)
     N = len(dimensions)
-    if N == 0:
-        return array([], dtype=dtype)
+    shape = (1,)*N
     res = empty((N,)+dimensions, dtype=dtype)
     for i, dim in enumerate(dimensions):
```

Comment:
Split the line because apparently we still worry about 80 chars sometimes. Think zipping is a little less obvious though, and I think that would be clearer as `...` than `:`